docs/release-notes/NuGet-6.3.md

```diff
@@ -0,0 +1,125 @@
+---
+title: NuGet 6.3 Release Notes
+description: Release notes for NuGet 6.3 including new features, bug fixes, and DCRs.
+author: <GithubAlias>
+ms.author: <MicrosoftAlias>
+ms.date: 8/2/2022
+ms.topic: conceptual
+---
+
+# NuGet 6.3 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**<NuGetVersion>**](https://nuget.org/downloads) | [Visual Studio <VSYear> version <VSVersion>](https://visualstudio.microsoft.com/downloads/) | [<SDKVersion>](https://dotnet.microsoft.com/download/dotnet-core/<SDKMajorMinorVersionOnly>)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio <VSYear> with.NET Core workload
+
+## Summary: What's New in 6.3
+
+* Implement "consuming pdbs from packages in PackageReference" - [#11352](https://github.com/NuGet/Home/issues/11352)
+
+* Create experimentation metrics to measure engagement of transitive dependencies - [#10759](https://github.com/NuGet/Home/issues/10759)
+
+* Enhance the experimentation infrastructure in NuGet code to support transitive dependencies - [#10758](https://github.com/NuGet/Home/issues/10758)
+
+* Allow to user to input custom (floating) versions through the PM UI - [#9829](https://github.com/NuGet/Home/issues/9829)
+
+* [Feature] Error on multiple entries for the same package ids in Directory.Packages.props - [#9467](https://github.com/NuGet/Home/issues/9467)
+
+### Issues fixed in this release
+
+**DCRs:**
+
+* [DCR]: Only cancel VS cred provider requests if VS is closing - [#11970](https://github.com/NuGet/Home/issues/11970)
+
+* Remove net45 and net40 targets from the NuGet build - [#11830](https://github.com/NuGet/Home/issues/11830)
```

Comment:
I think this should be at the bottom of the big changes. 

Maybe we reword it as `X, Y, Z no longer support net45 or net40` where X, Y and Z are the exact packages.

Reply:
+1. love this wording.

Reply:
@martinrrm this should be in what's new.

Reply:
https://github.com/NuGet/NuGet.Client/pull/4653/files has a number of them. Maybe just a few of them and then say "etc".